cmd/topology-aware/policy/topology-aware-policy.go

```diff
@@ -425,13 +426,13 @@ func (p *policy) reallocateResources(containers []cache.Container, pools map[str
 
 		grant, err := p.allocatePool(c, pools[c.GetCacheID()])
 		if err != nil {
-			errors = multierror.Append(errors, err)
+			errs = errors.Join(errs, err)
 		} else {
 			p.applyGrant(grant)
 		}
 	}
 
-	if err := multierror.New(errors); err != nil {
+	if err := errors.Join(errs); err != nil {
```

Comment:
Why can't this be just a simple `return err` here ? I mean, if we have collected no errors so far (IOW, all `err`s, in the above loop were `nil`, then errs should be nil at this point. The manual says

```
Join returns an error that wraps the given errors. Any nil error values are discarded. Join returns nil if errs contains no non-nil values.
```

AFAICT, `errors.Join()` serves no purpose if used as above.

Reply:
do'h, you are right. I didn't manually convert them one by one and that's why I probably missed it. So, I agree it doesn't make sense here. I will fix it.